_includes/gettingstarted.html

```diff
@@ -62,7 +65,9 @@ <h4 class="subheading">Write our steps</h4>
                                 </div>
                                 <div class="timeline-body">
                                     <p class="text-muted">Every line inside a Scenario is called a step. You have to provide a python function for every step in order to execute some code. Let's create a new "radish" directory and write our first step.py file there:
-                                        <pre style="width: 500px; text-align: left; margin-left: -100px"># -*- coding: utf-8 -*-
+                                        <pre style="width: 500px; text-align: left; margin-left: -100px">
+                                            <code class="Python">
+                                            # -*- coding: utf-8 -*-
```

Comment:
Inside `<code>`-tags the indentation actually matters. That's why this line will be indented in a weird way. 
Also pay attention to leading and trailing new/blank lines.